roles/confluent.ksql/tasks/main.yml

```diff
@@ -84,6 +84,23 @@
     owner: "{{ksql_user}}"
     mode: 0750
 
+- name: Create RocksDB Directory
```

Comment:
if this is going to be /tmp then it shouldnt be owned by ksql user
the important piece is that it is writeable by all users